### ISSUE TITLE
Most of collectible images are not fetched

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#21826]
Recently most of collectible images stopped being fetched. Issue occurs both on mobile and Desktop. cc @saledjenic 

[Status-debug-logs - 2024-12-16T120554.206.zip](https://github.com/user-attachments/files/18148724/Status-debug-logs.-.2024-12-16T120554.206.zip)

There are lot of errors such as:

`2024-12-16T09:33:14.438Z ERROR collectibles/manager.go:288 FetchAllAssetsByOwner failed for {"provider": "alchemy", "chainID": "10", "error": "Get \"https://opt-mainnet.g.alchemy.com/nft/v3/jSbhnlFGtfmwoPtGQKkSj9olKuxnbj6N/getNFTsForOwner?owner=0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be&withMetadata=true\": context canceled"}
2024-12-16T09:33:14.438Z WARN circuitbreaker/circuit_breaker.go:181 hystrix error {"provider": "alchemy10", "error": "Get \"https://opt-mainnet.g.alchemy.com/nft/v3/jSbhnlFGtfmwoPtGQKkSj9olKuxnbj6N/getNFTsForOwner?owner=0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be&withMetadata=true\": context canceled"}
2024-12-16T09:33:14.438Z ERROR collectibles/manager.go:288 FetchAllAssetsByOwner failed for {"provider": "alchemy", "chainID": "42161", "error": "Get \"https://arb-mainnet.g.alchemy.com/nft/v3/WM8_42w2_Oi1aNLxMwOeVu-x3zNTSl6H/getNFTsForOwner?owner=0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be&withMetadata=true\": context canceled"}
2024-12-16T09:33:14.438Z WARN circuitbreaker/circuit_breaker.go:181 hystrix error {"provider": "alchemy42161", "error": "Get \"https://arb-mainnet.g.alchemy.com/nft/v3/WM8_42w2_Oi1aNLxMwOeVu-x3zNTSl6H/getNFTsForOwner?owner=0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be&withMetadata=true\": context canceled"}
2024-12-16T09:33:14.438Z ERROR collectibles/manager.go:288 FetchAllAssetsByOwner failed for {"provider": "alchemy", "chainID": "1", "error": "Get \"https://eth-mainnet.g.alchemy.com/nft/v3/-aPgV9xyRDkinYHh4JuuepI9DdT5cXrv/getNFTsForOwner?owner=0xFC6327a092f6232e158a0Dd1d6d967a2e1c65cD5&withMetadata=true\": context canceled"}
2024-12-16T09:33:14.438Z WARN circuitbreaker/circuit_breaker.go:181 hystrix error {"provider": "alchemy1", "error": "Get \"https://eth-mainnet.g.alchemy.com/nft/v3/-aPgV9xyRDkinYHh4JuuepI9DdT5cXrv/getNFTsForOwner?owner=0xFC6327a092f6232e158a0Dd1d6d967a2e1c65cD5&withMetadata=true\": context canceled"}
2024-12-16T09:33:14.438Z ERROR collectibles/manager.go:288 FetchAllAssetsByOwner failed for {"provider": "openseaV2", "chainID": "42161", "error": "Get \"https://api.opensea.io/v2/chain/arbitrum/account/0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be/nfts?limit=50\": context canceled"}
2024-12-16T09:33:14.439Z ERROR collectibles/manager.go:306 FetchAllAssetsByOwner failed for {"chainID": "42161", "error": "rarible42161.error: context canceled, alchemy42161.error: context canceled, openseaV242161.error: Get \"https://api.opensea.io/v2/chain/arbitrum/account/0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be/nfts?limit=50\": context canceled"}
2024-12-16T09:33:14.439Z ERROR collectibles/commands.go:293 failed loadOwnedCollectiblesCommand {"chain": 42161, "account": "0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be", "page": 0, "error": "rarible42161.error: context canceled, alchemy42161.error: context canceled, openseaV242161.error: Get \"https://api.opensea.io/v2/chain/arbitrum/account/0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be/nfts?limit=50\": context canceled"}
2024-12-16T09:33:14.439Z DEBUG collectibles/commands.go:364 end loadOwnedCollectiblesCommand {"chain": 42161, "account": "0xBCE36f66A8cd99f1D6489CB9585146e3f3b893be", "in": "337.543646ms"}
2024-12-16T09:33:14.439Z ERROR collectibles/manager.go:288`

**MOBILE:**

https://github.com/user-attachments/assets/bfd672bc-a1f9-48d3-9af5-c7cad3f472f1

**DESKTOP:**

https://github.com/user-attachments/assets/0c0820c2-2825-447e-b83f-01304aad684b

- Status version: nightly 
- Operating System: Android, iOS, Desktop Mac OS

**Comment by ilmotta** [comment]
Just sharing a recent experience here.

From the latest Desktop 2.32.0 and Mobile 2.32.0, the collectible `Status-CC` always appears correctly in Desktop, but over Mobile (iPhone) I see the "Unsupported file" placeholder. So a bit different from this issue, the problem seems to be on Mobile (at least for this `Status-CC` collectible). I have an instance of Desktop 2.30 (much older) and the collectible `Status-CC` also appears correctly.

<img src="https://github.com/user-attachments/assets/44fdddfc-87f7-47d7-a460-5e6ecabb8fb3" width="200" />

**Comment by pavloburykh** [comment]
> Just sharing a recent experience here.
> 
> From the latest Desktop 2.32.0 and Mobile 2.32.0, the collectible `Status-CC` always appears correctly in Desktop, but over Mobile (iPhone) I see the "Unsupported file" placeholder. So a bit different from this issue, the problem seems to be on Mobile (at least for this `Status-CC` collectible). I have an instance of Desktop 2.30 (much older) and the collectible `Status-CC` also appears correctly.
> 
> <img alt="" width="200" src="https://private-user-images.githubusercontent.com/46027/401238697-44fdddfc-87f7-47d7-a460-5e6ecabb8fb3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc5NzkzOTAsIm5iZiI6MTczNzk3OTA5MCwicGF0aCI6Ii80NjAyNy80MDEyMzg2OTctNDRmZGRkZmMtODdmNy00N2Q3LWE0NjAtNWU2ZWNhYmI4ZmIzLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEyN1QxMTU4MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjJjZjcxMTEyMDJmYTFlYjdlYzAxNjAyNTAwMzE0ZTc0ZDliNTUwYjhkMWMyMmI1ZmQzNDExMWVlNTRlNzcyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Lzpx1SXYV96XFKGo-SoPM4wO1xEUu7qOOnuTEGpKO78">

At the same time, `Status-CC` collectible image is not loaded on Opensea as well https://opensea.io/assets/optimism/0x21f6f5cb75e81e5104d890d750270ed6538c50cb/34

Maybe there is some issue on Desktop side, related to the process of creating collectibles. For some reason the images are displayed only on Desktop itself, but not on mobile or opensea. 

I am closing current issue as the initial bug is not reproducible anymore. @ilmotta in case you think I still need to log separately the issue reported by you, please let me know. But at this point I am not sure there is an issue on mobile side.

**Comment by ilmotta** [comment]
> I am closing current issue as the initial bug is not reproducible anymore. @ilmotta in case you think I still need to log separately the issue reported by you, please let me know. But at this point I am not sure there is an issue on mobile side.

Thanks @pavloburykh. If it was just the image it would be more acceptable, but in my case, and maybe for other people, I can't use any of the Status community private channels, even though I have the rights for that.

@anastasiyaig have you crossed with this bug before? If you have the `StatusCC` collectible, using mobile app 2.32, can you see and interact with all private channels in the Status community?

**Comment by anastasiyaig** [comment]
I dont know the root cause of missing images. I checked mobile app from app store, and it shows me unknown image when the image of a token is present on OpenSea. 

https://github.com/user-attachments/assets/9a2b5b58-479c-44d4-9f2f-1dbf591511b2

at the same time, 2.32 on desktop shows icons for the token above and community tokens as well

![Image](https://github.com/user-attachments/assets/f389a433-42e5-4590-b1f6-bf1b1514663f)
![Image](https://github.com/user-attachments/assets/363572ca-b96e-4000-91a7-1a5d55fa0b61)
![Image](https://github.com/user-attachments/assets/04dbe442-4477-4ac6-83ec-eb0be7cd88ed)

also, at the same time, i was not able to see community tokens fetched (displayed) in my wallet on mobile, maybe it takes more time to receive them, i dont know

**Comment by pavloburykh** [comment]
> I dont know the root cause of missing images. I checked mobile app from app store, and it shows me unknown image when the image of a token is present on OpenSea.

Hey @anastasiyaig! Do you know anything about channel permission problem raised by Icaro? 

> ...in my case, and maybe for other people, I can't use any of the Status community private channels, even though I have the rights for tha
@anastasiyaig have you crossed with this bug before? If you have the StatusCC collectible, using mobile app 2.32, can you see and interact with all private channels in the Status community?

**Comment by anastasiyaig** [comment]
@pavloburykh sorry , i might not have been clear, let me rephrase. On mobile app, 2.32 version from apple store, after i imported account that is a member of status community, **i can't see private channels** , neither i can see community tokens in my wallet.

At the same time, same actions done on desktop client, allow me to see private channels, community and community tokens in wallet.

**Comment by anastasiyaig** [comment]
i think the issue with access to private channels could be filed in separate ticket and investigated. I bet it is because of control node being offline or not distributing updates. @ilmotta @pavloburykh pls tag me in the ticket if you decide to open one, could be in any of repos

**Comment by pavloburykh** [comment]
> i think the issue with access to private channels could be filed in separate ticket and investigated. I bet it is because of control node being offline or not distributing updates. [@ilmotta](https://github.com/ilmotta) [@pavloburykh](https://github.com/pavloburykh) pls tag me in the ticket if you decide to open one, could be in any of repos

Thank you @anastasiyaig! Will let you know once we investigate and log it. @ilmotta in case you are easily reproducing this issue on your account, would you mind log it and attach logs?

**Comment by pavloburykh** [comment]
@ilmotta @anastasiyaig I have reproduced the issue on both Desktop and Mobile, so I logged it in Desktop repo https://github.com/status-im/status-desktop/issues/17149

**Comment by anastasiyaig** [comment]
okay replied

**Comment by ilmotta** [comment]
> [@ilmotta](https://github.com/ilmotta) in case you are easily reproducing this issue on your account, would you mind log it and attach logs?

Sure @pavloburykh I'll try to bring logs more often in my bug reports. When it comes to personal profiles unfortunately I can't attach my logs directly (need to manually filter out or find possible culprits and share only those).

**Comment by pavloburykh** [comment]
> > [@ilmotta](https://github.com/ilmotta) in case you are easily reproducing this issue on your account, would you mind log it and attach logs?
> 
> Sure [@pavloburykh](https://github.com/pavloburykh) I'll try to bring logs more often in my bug reports. When it comes to personal profiles unfortunately I can't attach my logs directly (need to manually filter out or find possible culprits and share only those).

No worries @ilmotta! I have already reproduced and logged the issue with all necessary logs https://github.com/status-im/status-desktop/issues/17149  As it turned out reproduction was pretty straightforward.
